great_expectations/expectations/metrics/column_map_metrics/column_values_not_match_like_pattern_list.py

```diff
@@ -3,14 +3,14 @@
 from great_expectations.execution_engine.sqlalchemy_execution_engine import (
     SqlAlchemyExecutionEngine,
 )
-from great_expectations.expectations.metrics.import_manager import sa
 from great_expectations.expectations.metrics.map_metric_provider import (
     ColumnMapMetricProvider,
     column_condition_partial,
 )
 from great_expectations.expectations.metrics.util import (
     get_dialect_like_pattern_expression,
 )
+from great_expectations.optional_imports import sqlalchemy as sa
```

Comment:
Just wanted to say how much I like the `optional_imports`. It makes this whole (very repetitive) logic much more elegant.